go/vt/vttablet/tabletserver/tabletenv/tabletenv.go

```diff
@@ -70,6 +70,8 @@ var (
 	Warnings = stats.NewCounters("Warnings", "ResultsExceeded")
 	// Unresolved tracks unresolved items. For now it's just Prepares.
 	Unresolved = stats.NewCounters("Unresolved", "Prepares")
+	// Consolidations tracks how many queries were consolidated
+	Consolidations = stats.NewCounters("Consolidations", "qFetch")
```

Comment:
I think this is redundant. WaitStats already reports this number (which is what the test uses).

Reply:
aha! I see it now. I will rollback this portion of the change. Thanks!

Reply:
Fixed! Thanks for the pointer to the waitstats